Samples/AppLifecycle/Instancing/cpp-winui-packaged/CppWinUiDesktopInstancing/App.xaml

```diff
@@ -1,6 +1,4 @@
-﻿<!-- Copyright (c) Microsoft Corporation.
- Licensed under the MIT License. -->
```

Comment:
why was the license removed?

Reply:
No idea.  I added it back in.

Reply:
The license was removed because it is failing compilation.   Xaml Internal Error error WMC9999: Data at the root level is invalid. Line 1, position 2.

Build is successful when the build is removed.  I'll try and see what is wrong.

Reply:
Fixed the issue.  Somehow , one of those characters with 2 spaces in them was added to the start of the comment.